integration/Rakefile

```diff
@@ -1,10 +1,12 @@
 require 'rototiller'
+require 'fileutils'
 
 namespace :ci do
   namespace :test do
 
     desc 'Tests at the component level for the pe-r10k project'
     task :component => [:check_pe_r10k_env_vars] do
+      FileUtils.mkdir_p 'configs'
```

Comment:
Should this go in the `beaker_hostgenerator` task?

Reply:
updated!